benchmarks/benchmarks/bench_creation.py

```diff
@@ -13,7 +13,8 @@ class MeshGrid(Benchmark):
     timeout = 10
 
     def setup(self, size, ndims, ind, ndtype):
-        self.grid_dims = [(np.random.ranf(size)).astype(ndtype) for
+        rnd = np.random.RandomState(1864768776)
```

Comment:
If we are already touching these, should we use the recommended
```
rng = np.random.default_rng(1864768776)
rng.random(size, dtype)
```

Reply:
My reading of NEP 19 is that we're supposed to use `np.random.RandomState` for stable, reproducible RNG sequences across numpy verisons, which is why I used it explicitly in this PR. I guess it's been quite a while since NEP 19 - is that no longer the best recommendation?

Reply:
@ngoldbaum you're correct here. The testing use case (which applies to benchmarking too) for `RandomState` hasn't changed.